### ISSUE TITLE
Update release

### ISSUE THREAD

**Issue by anddam** [oanda/iOSNetworkingWithOandaApi#2]
Please provide an updated ready-to-use framework bundle, stripped of the unnecessary parts in the github project's releases.

The single tarball in releases is from December 2012.

**Comment by museyoucoulduse** [comment]
Yeah, please update this repo and possibly make it Swift 2. Here is some example: https://github.com/bm842/OandaRest
